<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Текстовый редактор</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="mFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="mOpen"/>
    <addaction name="mSave"/>
    <addaction name="mSaveAs"/>
    <addaction name="separator"/>
    <addaction name="mExit"/>
   </widget>
   <widget class="QMenu" name="mHelp">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="mReference"/>
   </widget>
   <addaction name="mFile"/>
   <addaction name="mHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="mSave">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="mOpen">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="mSaveAs">
   <property name="text">
    <string>Сохранить как...</string>
   </property>
  </action>
  <action name="mExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="mReference">
   <property name="text">
    <string>Справка</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
